jwst/flatfield/flat_field.py

```diff
@@ -434,7 +434,7 @@ def nirspec_fs_msa(output_model, f_flat_model, s_flat_model, d_flat_model, dispa
         flat_data_squared = slit_flat.data ** 2
         slit.var_poisson /= flat_data_squared
         slit.var_rnoise /= flat_data_squared
-        slit.var_flat = slit.data ** 2 / flat_data_squared * slit_flat.err ** 2
```

Comment:
If `flat_data_squared` is no longer used anywhere, the computation of it could be removed.

Reply:
It's still used in the two lines just above.

Reply:
D'oh!  Of course. Never mind ...